1.0/docs/devguide/templates.md

```diff
@@ -150,6 +148,9 @@ function defined on the host element.
 By default, the `filter` and `sort` functions only run when the array itself
 is mutated (for example, by adding or removing items).
```

Comment:
How about:

"when ..." =>
"when one of the following occurs:

*   The array itself is mutated (for example, by adding or removing items).
*   The `filter` or `sort` function is changed.